examples/spreadsheet.md

```diff
@@ -191,20 +204,74 @@ table.addEventListener("keyup", (event) => {
         highlight(target);
     }
 });
+
+table.addEventListener("keydown", (event) => {
+    const target = document.activeElement;
+    switch (event.keyCode) {
+        // left arrow
+        case 37:
+            step(target, -1, 0);
+            break;
+        // up arrow
+        case 38:
+            step(target, 0, -1);
+            break;
+        // right arrow
+        case 39:
+            step(target, 1, 0);
+            break;
+        // down arrow
+        case 40:
+            step(target, 0, 1);
+            break;
+    }
+});
 ```
 
 This also makes use of `increment()`, which uses some simple metadata-math
 to look up the cell in the next row of this column, and `focus()` it.
 
 ```javascript
-function increment(active_cell) {
+// TODO split function
+function step(active_cell, dx, dy) {
+    // TODO - this shouldn't be!
+    const scrollStep = 2; // if scroll * 2 > num rows or columns theres a problem
     const meta = table.getMeta(active_cell);
-    const rows = Array.from(table.querySelectorAll("tbody tr"));
-    const next_row = rows[meta.y - meta.y0 + 1];
-    if (next_row) {
-        const td = next_row.children[meta.x + 1];
-        td.focus();
+
+    if (meta.x + dx < NUM_COLUMNS && 0 <= meta.x + dx) {
+        xIndex = meta.x + dx;
+    }
+    if (dx !== 0) {
+        // scroll x axis if necessary
+        const columns = table.querySelectorAll("regular-table tbody tr:first-child td");
```

Comment:
Let's add `x1` and `y1` to `MetaData` instead of this please

Reply:
Is this what you have in mind, @texodus  -> https://github.com/jpmorganchase/regular-table/pull/55